plone/protect/utils.py

```diff
@@ -122,5 +131,22 @@ def safeWrite(obj, request=None):
     try:
         if obj._p_oid not in request.environ[SAFE_WRITE_KEY]:
             request.environ[SAFE_WRITE_KEY].append(obj._p_oid)
+        btree_types = (
```

Comment:
Sorry for me being picky, but please define the tuple outside of the function (on module level), its kind of static and does not need to be initialized on very call, right?

Reply:
Right.